### PR TITLE
Added l6174.is-a.dev

### PR DESCRIPTION
**Pull request by sekiryl**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

1. Profile Website: https://l6174.github.io
2. Startpage: https://l6174.github.io/startpage
3. Search: https://search.l6174.repl.co

### CODE REVIEW

File: domains/search.l6174.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "My Self-Hosted Search Engine",
+    "repo": "https://github.com/l6174/search",
+    "owner": {
+        "username": "l6174",
+        "email": "l.lawliet6174@gmail.com"
+    },
+    "record": {
+        "CNAME": "8b8a7371-0cc1-4815-affb-f53446c7b444.id.repl.co"
```
Comment (WilliamDavidHarrison) [inline domains/search.l6174.json:9]: Denied as it has a similar name to Google.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/search.l6174.json (hunk 1 above)
Comment (l6174) [reply domains/search.l6174.json:9]: Bro, It isn't on my control. The developers choosed to keep that name. The repository which I cloned is named "search". You can checkout this repository https://github.com/benbusby/whoogle-search It has MIT License.

Comment (WilliamDavidHarrison) [reply domains/search.l6174.json:9]: @l6174 It's the name of the project, not who made named it.

Comment (l6174) [reply domains/search.l6174.json:9]: @WilliamDavidHarrison I don't get it? What I am where I am supposed to change the name? My cloned repository? Or replit repository?

Comment (WilliamDavidHarrison) [reply domains/search.l6174.json:9]: > @WilliamDavidHarrison I don't get it? Where I am supposed to change the name? My cloned repository? Or replit repository?

Where ever the website will show.

Comment (l6174) [reply domains/search.l6174.json:9]: > > @WilliamDavidHarrison I don't get it? Where I am supposed to change the name? My cloned repository? Or replit repository?
> 
> Where ever the website will show.

Nevermind, I am deleting search file. I will use https://search.l6174.repl.co .

Comment (l6174) [reply domains/search.l6174.json:9]: See this:
https://search.thomasc.is-a.dev/
It is using same "Whoogle" hosted on repl.
IDK Why I can't?

Comment (WilliamDavidHarrison) [reply domains/search.l6174.json:9]: I didn't merge that pull request. I'm not sure why it is there.

Comment (l6174) [reply domains/search.l6174.json]: Deleted. So, Basically we can't use whoogle with .is-a.dev. Anyways, I have deleted the file you can check further and merge. Thanks!

**Review by wdhdev — APPROVED** [review]

File: domains/search.l6174.json (hunk 1 above)
Comment (WilliamDavidHarrison) [reply domains/search.l6174.json]: If you want a domain for it, Free Domains will let you. https://github.com/free-domains/register

**Comment by WilliamDavidHarrison** [comment]
> @WilliamDavidHarrison i'm not sure whether their 'whoogle' is impersonating/copyrighted trademark.

It's a similar name to Google though.

**Pull request merged**